doc/tutorials/intermediate/index.md

```diff
@@ -67,6 +67,7 @@ Embark on a deeper exploration of supplementary topics to further hone your Pane
 Now that you've mastered the more advanced concepts of Panel, it's time to put your skills to the test:
 
 - **Create an Interactive Report:** Elevate the interactivity of your reports through embedding.
+- **[Create a Form](create_form.md):** Create a form to sign up for a news letter.
```

Comment:
With Reflex a Form is the first thing they teach/ show their users. But they don't have to manually add validation. Its built into the components.